udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -20,13 +20,55 @@ async function deleteCommentData() {
   await getCommentData;
   window.location.reload();
 }
+
 /** Creates an <li> element containing text. */
 function createCommentListItem(inputText) {
   const listElement = document.createElement('li');
   listElement.innerText = inputText;
   return listElement;
 }
 
+/** Creates a map to show University. */
```

Comment:
/** Creates a map with a marker at UC San Diego. */

Would be more specific.

Reply:
Done